configs/config_params.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+
+##USER NOTE: Certain fields are commented out as they are
+## not mandatory. Uncomment relevant fields based on your
+## use case.
+
+
+#Learning args
+clip_grad: 0 
+lr: 0.0001 
+epochs: 5
+afunc: relu 
+mse_weight: 0.001 
+pearson_weight: 1
+poisson_weight: 0
+# Experiment args
+label: 'AtacWorks'
+out_home: './Cache'
+train: False
+infer: False
+resume: False
+eval: False
+infer_files: ""
+intervals_file: ""
+sizes_file: ""
+infer_threshold:
+reg_rounding: 0
+cla_rounding: 3
+batches_per_worker: 16
+gen_bigwig: False
+weights_path:
+result_fname: 'infer_results.h5'
+deletebg: False
+# Training args
+task: 'both'
+train_files: ""
+print_freq: 10
+bs: 8
+num_workers: 4
+checkpoint_fname: ""
+save_freq: 1
+# Dataset args
+pad:
```

Comment:
I'm surprised leaving it empty works instead of requiring a "None" string here. I think it would be better to instead pass "None" as a string in cases where we explicitly want None

Reply:
done.